.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -21,3 +21,12 @@ Als [Rolle] möchte ich [Funktionalität], damit ich [Grund].
 - [ ] Die User Story ist klar und verständlich aus der Perspektive der betroffenen Persona formuliert.
 - [ ] Vollständige und klare Akzeptanzkriterien sind definiert.
 - [ ] Die Aufwandsschätzung für das Issue wurde durchgeführt und dokumentiert.
+
+---
+**Definition of Done (DoD):**
+
+- [ ] Unittest gepflegt
+- [ ] Dokumentation ist gepflegt
```

Comment:
according to the issue, these two tasks should be part of the pull request template:

```
Punkte für Pull Request:

    Unittest gepflegt
    Dokumentation ist gepflegt
```